### PR TITLE
r/s3_bucket: read-only `cors_rule` argument

### PR DESCRIPTION
**Pull request by anGie44**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #4418
Relates #20433

Output from acceptance testing:

```
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_empty (15.29s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_multipleTags (170.44s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefix (172.94s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_prefixAndTags (172.65s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_remove (170.08s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithFilter_singleTag (171.93s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_default (105.50s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_empty (15.87s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_WithStorageClassAnalysis_full (103.85s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_basic (104.25s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_removed (159.57s)
--- PASS: TestAccS3BucketAnalyticsConfiguration_updateBasic (246.59s)

--- PASS: TestAccS3BucketDataSource_basic (91.33s)
--- PASS: TestAccS3BucketDataSource_website (90.21s)

--- PASS: TestAccS3BucketIntelligentTieringConfiguration_Filter (389.48s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (103.00s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (89.11s)

--- PASS: TestAccS3BucketInventory_basic (104.17s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (102.87s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (104.04s)

--- PASS: TestAccS3BucketMetric_basic (102.64s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (15.92s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (170.24s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (169.37s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (168.57s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (171.03s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (172.36s)

--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (118.76s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (101.66s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (120.20s)
--- PASS: TestAccS3BucketNotification_queue (131.90s)
--- PASS: TestAccS3BucketNotification_topic (103.45s)
--- PASS: TestAccS3BucketNotification_update (188.60s)

--- PASS: TestAccS3BucketObjectDataSource_allParams (90.81s)
--- PASS: TestAccS3BucketObjectDataSource_basic (87.96s)
--- PASS: TestAccS3BucketObjectDataSource_basicViaAccessPoint (91.47s)
--- PASS: TestAccS3BucketObjectDataSource_bucketKeyEnabled (91.67s)
--- PASS: TestAccS3BucketObjectDataSource_kmsEncrypted (91.23s)
--- PASS: TestAccS3BucketObjectDataSource_leadingSlash (162.83s)
--- PASS: TestAccS3BucketObjectDataSource_multipleSlashes (161.48s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOff (91.39s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOn (91.36s)
--- PASS: TestAccS3BucketObjectDataSource_readableBody (92.56s)
--- PASS: TestAccS3BucketObjectDataSource_singleSlashAsKey (89.23s)

--- PASS: TestAccS3BucketObject_acl (257.31s)
--- PASS: TestAccS3BucketObject_bucketBucketKeyEnabled (95.10s)
--- PASS: TestAccS3BucketObject_content (102.75s)
--- PASS: TestAccS3BucketObject_contentBase64 (89.50s)
--- PASS: TestAccS3BucketObject_defaultBucketSSE (91.40s)
--- PASS: TestAccS3BucketObject_empty (102.90s)
--- PASS: TestAccS3BucketObject_etagEncryption (104.33s)
--- PASS: TestAccS3BucketObject_ignoreTags (168.63s)
--- PASS: TestAccS3BucketObject_kms (101.94s)
--- PASS: TestAccS3BucketObject_metadata (256.36s)
--- PASS: TestAccS3BucketObject_noNameNoKey (24.76s)
--- PASS: TestAccS3BucketObject_nonVersioned (104.20s)
--- PASS: TestAccS3BucketObject_objectBucketKeyEnabled (89.23s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithNone (244.57s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithOn (169.19s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithNone (245.16s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithSet (321.28s)
--- PASS: TestAccS3BucketObject_source (103.68s)
--- PASS: TestAccS3BucketObject_sourceHashTrigger (175.28s)
--- PASS: TestAccS3BucketObject_sse (101.82s)
--- PASS: TestAccS3BucketObject_storageClass (401.97s)
--- PASS: TestAccS3BucketObject_tags (326.15s)
--- PASS: TestAccS3BucketObject_tagsLeadingMultipleSlashes (314.96s)
--- PASS: TestAccS3BucketObject_tagsLeadingSingleSlash (327.88s)
--- PASS: TestAccS3BucketObject_tagsMultipleSlashes (312.12s)
--- PASS: TestAccS3BucketObject_updateSameFile (165.02s)
--- PASS: TestAccS3BucketObject_updates (177.01s)
--- PASS: TestAccS3BucketObject_updatesWithVersioning (178.26s)
--- PASS: TestAccS3BucketObject_updatesWithVersioningViaAccessPoint (167.17s)
--- PASS: TestAccS3BucketObject_withContentCharacteristics (90.60s)

--- PASS: TestAccS3BucketObjectsDataSource_all (174.17s)
--- PASS: TestAccS3BucketObjectsDataSource_basic (174.32s)
--- PASS: TestAccS3BucketObjectsDataSource_basicViaAccessPoint (172.06s)
--- PASS: TestAccS3BucketObjectsDataSource_encoded (171.96s)
--- PASS: TestAccS3BucketObjectsDataSource_fetchOwner (171.97s)
--- PASS: TestAccS3BucketObjectsDataSource_maxKeys (171.67s)
--- PASS: TestAccS3BucketObjectsDataSource_prefixes (172.24s)
--- PASS: TestAccS3BucketObjectsDataSource_startAfter (171.08s)

--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (81.69s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (184.64s)
--- PASS: TestAccS3BucketOwnershipControls_basic (108.15s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (90.71s)

--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (371.89s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (279.19s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (295.98s)

--- PASS: TestAccS3BucketPolicy_basic (103.13s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (192.30s)

--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (80.24s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (105.45s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (255.34s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (252.56s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (92.07s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (250.18s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (248.65s)

--- PASS: TestAccS3BucketReplicationConfiguration_basic (545.58s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (491.16s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (483.62s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (141.65s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (365.32s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (361.77s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (408.90s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (403.37s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (401.47s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (400.26s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (398.13s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (364.55s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (231.34s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (405.89s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (318.90s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (398.18s)

--- PASS: TestAccS3BucketVersioning_MFADelete (65.70s)
--- PASS: TestAccS3BucketVersioning_basic (107.38s)
--- PASS: TestAccS3BucketVersioning_disappears (95.76s)
--- PASS: TestAccS3BucketVersioning_update (148.03s)

--- PASS: TestAccS3Bucket_Basic_acceleration (63.99s)
--- PASS: TestAccS3Bucket_Basic_basic (60.49s)
--- PASS: TestAccS3Bucket_Basic_emptyString (60.80s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (98.89s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (98.62s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (98.13s)
--- PASS: TestAccS3Bucket_Basic_generatedName (38.37s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (73.35s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (39.60s)
--- PASS: TestAccS3Bucket_Basic_requestPayer (59.05s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (52.43s)
--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (82.85s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (225.14s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (176.24s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (95.47s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (81.58s)
--- PASS: TestAccS3Bucket_Manage_objectLock (150.97s)
--- PASS: TestAccS3Bucket_Manage_versioning (158.13s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (83.50s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (74.14s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (256.48s)
--- PASS: TestAccS3Bucket_Replication_basic (310.01s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (76.05s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (167.12s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (169.75s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (227.19s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (225.34s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (298.63s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (115.31s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (166.33s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (155.92s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (163.25s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (76.86s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (139.88s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (62.99s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (58.65s)
--- PASS: TestAccS3Bucket_Security_grantToACL (92.25s)
--- PASS: TestAccS3Bucket_Security_logging (87.51s)
--- PASS: TestAccS3Bucket_Security_policy (91.50s)
--- PASS: TestAccS3Bucket_Security_updateACL (64.53s)
--- PASS: TestAccS3Bucket_Security_updateGrant (114.03s)
--- PASS: TestAccS3Bucket_Tags_basic (41.28s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (63.82s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (138.76s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (169.40s)
--- PASS: TestAccS3Bucket_Web_redirect (178.59s)
--- PASS: TestAccS3Bucket_Web_routingRules (111.00s)
--- PASS: TestAccS3Bucket_Web_simple (157.11s)

--- PASS: TestBucketName (0.00s)
--- PASS: TestBucketRegionalDomainName (0.00s)
--- PASS: TestWebsiteEndpoint (0.00s)
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
```

### CODE REVIEW

**Comment by anGie44** [comment]
Optionally preceded by #12141

File: internal/service/s3/bucket_test.go [hunk 1]
```diff
@@ -1062,169 +1062,6 @@ func TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled(t *testing.T) {
 	})
 }
 
-func TestAccS3Bucket_Security_corsUpdate(t *testing.T) {
```
Comment (anGie44) [inline internal/service/s3/bucket_test.go:1065]: These tests are (or will be) ported over to #12141

File: .changelog/22611.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```release-note:note
```
Comment (anGie44) [inline .changelog/22611.txt:1]:
```suggestion
```release-note:breaking-change
```

**Comment by anGie44** [comment]
Needs mention in upgrade guide

**Review by gdavison — APPROVED** [review]
Looks good! 🚀 

--- SKIP: TestAccS3BucketLogging_TargetGrantByEmail (0.00s)
--- SKIP: TestAccS3BucketReplicationConfiguration_existingObjectReplication (0.00s)
--- PASS: TestBucketName (0.00s)
--- PASS: TestBucketRegionalDomainName (0.00s)
--- PASS: TestWebsiteEndpoint (0.00s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithObjectLockEnabled (119.89s)
--- PASS: TestAccS3Bucket_Basic_forceDestroyWithEmptyPrefixes (122.47s)
--- PASS: TestAccS3Bucket_Basic_forceDestroy (125.99s)
--- PASS: TestAccS3BucketVersioning_disappears (126.98s)
--- PASS: TestAccS3BucketWebsiteConfiguration_disappears (134.84s)
--- PASS: TestAccS3BucketAccelerateConfiguration_basic (135.48s)
--- PASS: TestAccS3BucketWebsiteConfiguration_Redirect (136.20s)
--- PASS: TestAccS3BucketVersioning_MFADelete (136.59s)
--- PASS: TestAccS3BucketWebsiteConfiguration_RoutingRules_RedirectOnly (137.41s)
--- PASS: TestAccS3BucketVersioning_basic (138.42s)
--- PASS: TestAccS3BucketWebsiteConfiguration_basic (141.85s)
--- PASS: TestAccS3Bucket_Replication_schemaV2SameRegion (146.71s)
--- PASS: TestAccS3Bucket_Replication_withoutPrefix (223.45s)
--- PASS: TestAccS3Bucket_Manage_objectLock (225.25s)
--- PASS: TestAccS3Bucket_Replication_expectVersioningValidationError (113.93s)
--- PASS: TestAccS3BucketWebsiteConfiguration_update (242.03s)
--- PASS: TestAccS3BucketWebsiteConfiguration_RoutingRules_MultipleRules (245.31s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleExpirationEmptyBlock (124.54s)
--- PASS: TestAccS3Bucket_Manage_lifecycleRuleAbortIncompleteMultipartUploadDaysNoExpiration (138.48s)
--- PASS: TestAccS3BucketVersioning_update (350.86s)
--- PASS: TestAccS3Bucket_Replication_withoutStorageClass (229.47s)
--- PASS: TestAccS3Bucket_Basic_shouldFailNotFound (101.53s)
--- PASS: TestAccS3Bucket_Manage_versioningAndMfaDeleteDisabled (143.60s)
--- PASS: TestAccS3Bucket_Security_logging (150.32s)
--- PASS: TestAccS3Bucket_Manage_MfaDeleteDisabled (140.44s)
--- PASS: TestAccS3Bucket_Replication_twoDestination (239.94s)
--- PASS: TestAccS3Bucket_Manage_versioningDisabled (133.27s)
--- PASS: TestAccS3Bucket_Manage_lifecycleExpireMarkerOnly (236.43s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsNonEmptyFilter (243.15s)
--- PASS: TestAccS3Bucket_Replication_multipleDestinationsEmptyFilter (242.91s)
--- PASS: TestAccS3BucketWebsiteConfiguration_RoutingRules_ConditionAndRedirect (386.45s)
--- PASS: TestAccS3Bucket_Basic_keyEnabled (150.95s)
--- PASS: TestAccS3Bucket_Manage_lifecycleBasic (332.65s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenAES256IsUsed (134.27s)
--- PASS: TestAccS3Bucket_Manage_versioning (252.91s)
--- PASS: TestAccS3Bucket_Security_enableDefaultEncryptionWhenTypical (136.15s)
--- PASS: TestAccS3Bucket_Basic_generatedName (134.93s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAddAccessControlTranslation (391.01s)
--- PASS: TestAccS3Bucket_Replication_ruleDestinationAccessControlTranslation (391.58s)
--- PASS: TestAccS3Bucket_Basic_namePrefix (133.28s)
--- PASS: TestAccS3Bucket_Tags_basic (129.83s)
--- PASS: TestAccS3Bucket_Security_disableDefaultEncryptionWhenDefaultEncryptionIsEnabled (209.56s)
--- PASS: TestAccS3Bucket_Security_grantToACL (192.77s)
--- PASS: TestAccS3Bucket_Security_aclToGrant (199.35s)
--- PASS: TestAccS3Bucket_Security_updateACL (198.66s)
--- PASS: TestAccS3Bucket_Basic_requestPayer (204.08s)
--- PASS: TestAccS3Bucket_Basic_acceleration (204.30s)
--- PASS: TestAccS3Bucket_Basic_emptyString (100.99s)
--- PASS: TestAccS3Bucket_Basic_basic (101.22s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySSEByDefault_KMSWithMasterKeyID (102.04s)
--- PASS: TestAccS3Bucket_Security_updateGrant (274.30s)
--- PASS: TestAccS3Bucket_Security_policy (275.12s)
--- PASS: TestAccS3Bucket_Replication_RTC_valid (654.06s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySSEByDefault_KMSWithMasterKeyArn (100.03s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_disappears (88.78s)
--- PASS: TestAccS3Bucket_Tags_ignoreTags (183.05s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySSEByDefault_KMS (102.21s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySEEByDefault_AES256 (101.41s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_basic (100.03s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySSEByDefault_BucketKeyEnabled (187.24s)
--- PASS: TestAccS3BucketRequestPaymentConfiguration_Basic_Requester (103.83s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_BucketKeyEnabled (189.61s)
--- PASS: TestAccS3BucketRequestPaymentConfiguration_Basic_BucketOwner (115.87s)
--- PASS: TestAccS3BucketServerSideEncryptionConfiguration_ApplySSEByDefault_UpdateSSEAlgorithm (197.25s)
--- PASS: TestAccS3Bucket_Replication_basic (642.01s)
--- PASS: TestAccS3Bucket_Replication_schemaV2 (793.28s)
--- PASS: TestAccS3Bucket_Tags_withNoSystemTags (324.93s)
--- PASS: TestAccS3BucketRequestPaymentConfiguration_update (259.40s)
--- PASS: TestAccS3Bucket_Tags_withSystemTags (366.37s)
--- PASS: TestAccS3BucketReplicationConfiguration_disappears (99.40s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2SameRegion (228.64s)
--- PASS: TestAccS3BucketPublicAccessBlock_Disappears_bucket (35.95s)
--- PASS: TestAccS3BucketPublicAccessBlock_restrictPublicBuckets (121.68s)
--- PASS: TestAccS3BucketPublicAccessBlock_disappears (32.01s)
--- PASS: TestAccS3BucketPublicAccessBlock_ignorePublicACLs (101.98s)
--- PASS: TestAccS3BucketPublicAccessBlock_basic (35.48s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicPolicy (90.76s)
--- PASS: TestAccS3BucketPublicAccessBlock_blockPublicACLs (84.68s)
--- PASS: TestAccS3BucketPolicy_basic (21.51s)
--- PASS: TestAccS3BucketPolicy_policyUpdate (37.69s)
--- PASS: TestAccS3BucketOwnershipControls_Disappears_bucket (21.20s)
--- PASS: TestAccS3BucketOwnershipControls_Rule_objectOwnership (39.54s)
--- PASS: TestAccS3BucketOwnershipControls_disappears (22.08s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDoc (75.33s)
--- PASS: TestAccS3BucketOwnershipControls_basic (25.37s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutPrefix (388.99s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_policyDocNotPrincipal (93.86s)
--- PASS: TestAccS3BucketObjectsDataSource_fetchOwner (38.61s)
--- PASS: TestAccS3BucketObjectsDataSource_startAfter (43.36s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_tagFilter (392.97s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2DestinationMetrics (389.78s)
--- PASS: TestAccS3BucketObjectsDataSource_maxKeys (46.65s)
--- PASS: TestAccS3BucketObjectsDataSource_encoded (49.81s)
--- PASS: TestAccS3BucketReplicationConfiguration_schemaV2 (389.92s)
--- PASS: TestAccS3BucketReplicationConfiguration_withoutStorageClass (394.52s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicationTimeControl (384.25s)
--- PASS: TestAccS3BucketReplicationConfiguration_replicaModifications (394.90s)
--- PASS: TestAccS3BucketObjectsDataSource_prefixes (58.11s)
--- PASS: TestAccS3BucketObjectsDataSource_all (60.66s)
--- PASS: TestAccS3BucketObjectsDataSource_basicViaAccessPoint (67.73s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsNonEmptyFilter (375.54s)
--- PASS: TestAccS3BucketReplicationConfiguration_twoDestination (378.86s)
--- PASS: TestAccS3BucketPolicy_IAMRoleOrder_jsonEncode (179.49s)
--- PASS: TestAccS3BucketReplicationConfiguration_multipleDestinationsEmptyFilter (365.28s)
--- PASS: TestAccS3BucketObject_defaultBucketSSE (58.76s)
--- PASS: TestAccS3BucketObject_bucketBucketKeyEnabled (58.32s)
--- PASS: TestAccS3BucketObjectsDataSource_basic (81.00s)
--- PASS: TestAccS3BucketReplicationConfiguration_filter_andOperator (464.46s)
--- PASS: TestAccS3BucketObject_objectBucketKeyEnabled (60.07s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAddAccessControlTranslation (445.23s)
--- PASS: TestAccS3BucketReplicationConfiguration_configurationRuleDestinationAccessControlTranslation (434.12s)
--- PASS: TestAccS3BucketObject_ignoreTags (126.91s)
--- SKIP: TestAccS3BucketObject_nonVersioned (0.00s)
--- PASS: TestAccS3BucketReplicationConfiguration_basic (400.54s)
--- PASS: TestAccS3BucketObject_bucketObjectBucket (146.04s)
--- PASS: TestAccS3BucketObject_kms (86.64s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithOn (131.59s)
--- PASS: TestAccS3BucketObject_sse (90.49s)
--- PASS: TestAccS3BucketObject_metadata (185.24s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithNone (187.26s)
--- PASS: TestAccS3BucketObject_withContentCharacteristics (92.83s)
--- PASS: TestAccS3BucketObject_noNameNoKey (20.10s)
--- PASS: TestAccS3BucketObject_updatesWithVersioningViaAccessPoint (154.29s)
--- PASS: TestAccS3BucketObject_objectLockLegalHoldStartWithNone (195.89s)
--- PASS: TestAccS3BucketObject_contentBase64 (91.20s)
--- PASS: TestAccS3BucketObject_updatesWithVersioning (164.24s)
--- PASS: TestAccS3BucketObject_updateSameFile (154.47s)
--- PASS: TestAccS3BucketObject_etagEncryption (93.49s)
--- PASS: TestAccS3BucketObject_content (98.84s)
--- PASS: TestAccS3BucketObject_updates (161.72s)
--- PASS: TestAccS3BucketObject_source (100.77s)
--- PASS: TestAccS3BucketObject_objectLockRetentionStartWithSet (258.97s)
--- PASS: TestAccS3BucketObject_empty (96.47s)
--- PASS: TestAccS3BucketObject_acl (226.75s)
--- PASS: TestAccS3BucketObject_sourceHashTrigger (169.58s)
--- PASS: TestAccS3BucketObjectDataSource_singleSlashAsKey (83.01s)
--- PASS: TestAccS3BucketObjectLockConfiguration_disappears (88.60s)
--- PASS: TestAccS3BucketObject_tagsMultipleSlashes (275.38s)
--- PASS: TestAccS3BucketObject_tagsLeadingSingleSlash (274.81s)
--- PASS: TestAccS3BucketObject_tagsLeadingMultipleSlashes (275.79s)
--- PASS: TestAccS3BucketObjectLockConfiguration_basic (99.86s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOn (91.79s)
--- PASS: TestAccS3BucketObject_tags (281.93s)
--- PASS: TestAccS3BucketObjectDataSource_objectLockLegalHoldOff (91.73s)
--- PASS: TestAccS3BucketObjectDataSource_allParams (89.24s)
--- PASS: TestAccS3BucketObjectDataSource_bucketKeyEnabled (90.99s)
--- PASS: TestAccS3BucketObjectDataSource_kmsEncrypted (93.33s)
--- PASS: TestAccS3BucketObjectDataSource_readableBody (90.04s)
--- PASS: TestAccS3BucketMetric_withEmptyFilter (12.76s)
--- PASS: TestAccS3BucketObjectDataSource_basicViaAccessPoint (84.27s)
--- PASS: TestAccS3BucketObjectDataSource_basic (85.54s)
--- PASS: TestAccS3BucketObjectDataSource_multipleSlashes (146.26s)
--- PASS: TestAccS3BucketObjectDataSource_leadingSlash (146.04s)
--- FAIL: TestAccS3BucketLogging_update (9.06s)
--- PASS: TestAccS3BucketObjectLockConfiguration_update (169.24s)
--- PASS: TestAccS3BucketObject_storageClass (346.00s)
--- PASS: TestAccS3BucketNotification_Topic_multiple (97.23s)
--- PASS: TestAccS3BucketNotification_topic (95.02s)
--- PASS: TestAccS3BucketNotification_eventbridge (90.78s)
--- PASS: TestAccS3BucketNotification_LambdaFunctionLambdaFunctionARN_alias (116.10s)
--- PASS: TestAccS3BucketNotification_lambdaFunction (117.85s)
--- PASS: TestAccS3BucketNotification_queue (128.77s)
--- PASS: TestAccS3BucketMetric_basic (90.13s)
--- PASS: TestAccS3BucketLogging_disappears (90.27s)
--- PASS: TestAccS3BucketLogging_basic (100.07s)
--- PASS: TestAccS3BucketMetric_withFilterMultipleTags (155.23s)
--- PASS: TestAccS3BucketMetric_withFilterSingleTag (157.78s)
--- PASS: TestAccS3BucketNotification_update (186.35s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndMultipleTags (151.26s)
--- PASS: TestAccS3BucketMetric_withFilterPrefixAndSingleTag (155.99s)
--- PASS: TestAccS3BucketMetric_withFilterPrefix (154.57s)
--- PASS: TestAccS3BucketInventory_encryptWithSSEKMS (72.73s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_disappears (64.13s)
--- PASS: TestAccS3BucketInventory_encryptWithSSES3 (72.71s)
--- PASS: TestAccS3BucketInventory_basic (73.83s)
--- PASS: TestAccS3BucketDataSource_website (60.37s)
--- PASS: TestAccS3BucketIntelligentTieringConfiguration_basic (69.34s)
--- PASS: TestAccS3BucketLogging_TargetGrantByGroup (214.47s)
--- PASS: TestAccS3BucketLogging_TargetGrantByID (212.33s)
--- FAIL: TestAccS3BucketCorsConfiguration_update (28.68s)

The failures are intermittent

**Comment by anGie44** [comment]
Output of updated tests:
```
--- PASS: TestAccS3BucketCorsConfiguration_disappears (29.92s)
--- PASS: TestAccS3BucketCorsConfiguration_SingleRule (32.64s)
--- PASS: TestAccS3BucketCorsConfiguration_basic (33.14s)
--- PASS: TestAccS3BucketCorsConfiguration_MultipleRules (33.14s)
--- PASS: TestAccS3BucketCorsConfiguration_update (81.43s)
```

**Pull request merged**